indexer/indexer_tests/cities_boundaries_serdes_tests.cpp

```diff
@@ -0,0 +1,113 @@
+#include "testing/testing.hpp"
+
+#include "indexer/city_boundary.hpp"
+#include "indexer/cities_boundaries_serdes.hpp"
+#include "indexer/coding_params.hpp"
+
+#include "coding/reader.hpp"
+#include "coding/writer.hpp"
+
+#include "geometry/mercator.hpp"
+#include "geometry/point2d.hpp"
+
+#include <cstdint>
+#include <vector>
+
+using namespace indexer;
+using namespace m2;
+using namespace serial;
+using namespace std;
+
+namespace
+{
+using Boundary = vector<CityBoundary>;
+using Boundaries = vector<Boundary>;
+
+void TestEqual(BoundingBox const & lhs, BoundingBox const & rhs, double eps)
```

Comment:
TestAlmostEqual()? (Here and below).

Reply:
Not sure, why? The method has eps parameter, used only here, what's the reason to complicate namings?

Reply:
Ok. Let's leave it as TestEqual()